src/vs/platform/clipboard/common/clipboardService.ts

```diff
@@ -8,9 +8,22 @@ import { URI } from 'vs/base/common/uri';
 
 export const IClipboardService = createDecorator<IClipboardService>('clipboardService');
 
+// {{SQL CARBON EDIT}}
```

Comment:
Add comments here - and provide link to https://www.electronjs.org/docs/api/clipboard#clipboardwritedata-type